server/graphql/v2/input/RecurringExpenseInput.ts

```diff
@@ -0,0 +1,20 @@
+import { GraphQLInputObjectType, GraphQLNonNull } from 'graphql';
+
+import { DateString } from '../../v1/types';
+import { RecurringExpenseInterval } from '../enum/RecurringExpenseInterval';
+
+const RecurringExpenseInput = new GraphQLInputObjectType({
+  name: 'RecurringExpenseInput',
+  fields: () => ({
+    interval: {
+      type: new GraphQLNonNull(RecurringExpenseInterval),
+      description: 'The interval in which this recurring expense is created',
+    },
+    endsAt: {
+      type: DateString,
```

Comment:
We used this type in the expense flow, but it was a mistake. The right one is `import { GraphQLDateTime } from 'graphql-scalars';`